ssms/config/config.py

```diff
@@ -367,3 +376,39 @@ def boundary_config_to_function_params(config: dict) -> dict:
 model_config["ddm_mic2_leak_conflict_gamma_no_bias_no_lowdim_noise"] = model_config[
     "ddm_mic2_leak_conflict_gamma_no_bias"
 ].copy()
+
+
+def get_data_generator_config(approach) -> dict:
```

Comment:
slightly different from the model configs, where the actual choices of entries are sort of meant to be used as is (let's say 99/1 ratio), these data-generator configs, are more 'defaults' in the sense that you are more or less expected to make some changes. 

so maybe here `get_basic...` or `get_default...` could be a better naming convention?